### DISCUSSION TITLE
insights operator is Degraded

### DISCUSSION THREAD

**Discussion by baturorkun** [okd-project/okd#626] [Q&A] [answered]
my insights operator is Degraded

Unable to report: unable to build request to connect to Insights server: Post "https://cloud.redhat.com/api/ingress/v1/upload": x509: certificate is valid for *.mydomain.com.tr, mydomain.com.tr, not cloud.redhat.com

my main domain: mydomain.com.tr 
cluster name : okd
API : api.okd.mydomain.com.tr

I found  "https://access.redhat.com/solutions/5807481"

But after applied this solution, 5 extra operators was degraded too

**Comment by baturorkun** [comment]
I revoked the patch ( deleted ca name ) but my system has  still problems
okd-q59nn-master-1         NotReady,SchedulingDisabled   master   12d   v1.19.2+f173eb4-1049   192.168.2.151   <none>        Fedora CoreOS 33.20210117.3.2   5.10.12-200.fc33.x86_64   cri-o://1.19.1

**Comment by vrutkovs** [comment]
Please attach (or upload to the public file sharing service) [must-gather](https://docs.okd.io/latest/support/gathering-cluster-data.html#support_gathering_data_gathering-cluster-data) archive

**Reply by baturorkun** [reply]
hi  vadim
I upload the log to my drive and share with you

thanks a lot

**Reply by vrutkovs** [reply]
That's great, but I'm not a dedicated OKD support person, the community should have access to it to have a look

**Reply by baturorkun** [reply]
I see. But  I noticed that the logs includes domain information  and maybe some the other information I don't know. I am not sure to share to the public? :(

**Comment by baturorkun** [comment]
I found  that issue: https://github.com/openshift/okd/issues/71

my domain is like "okd.example.com"

@jdhirst mentioned removing *.okd records.

https://github.com/openshift/okd/issues/71#issuecomment-583017825

**Comment by baturorkun** [comment, marked as answer]
I deleted my cluster named  "OKD" and created a different name. The insight operator was working.
